reports/scorecard/report.md

```diff
@@ -4,79 +4,79 @@
+| [cisco-ospo/.github](https://github.com/cisco-ospo/.github) | [a645dde](https://github.com/cisco-ospo/.github/commit/a645dde125399520823aa48501515d12684c66bb) | 5.9 | 2024-03-18T21:41:02Z | 0 / [Details](https://ossf.github.io/scorecard-visualizer/#/projects/github.com/cisco-ospo/.github/compare/a645dde125399520823aa48501515d12684c66bb/a645dde125399520823aa48501515d12684c66bb) | [View](https://ossf.github.io/scorecard-visualizer/#/projects/github.com/cisco-ospo/.github/commit/a645dde125399520823aa48501515d12684c66bb) | [Fix it](https://app.stepsecurity.io/securerepo?repo=cisco-ospo/.github) |
+| [cisco-ospo/oss-template](https://github.com/cisco-ospo/oss-template) | [99689cc](https://github.com/cisco-ospo/oss-template/commit/99689cce7148816cbaae4cd0d75e0de82f434869) | 6.3 | 2024-06-15T01:48:23Z | 0 / [Details](https://ossf.github.io/scorecard-visualizer/#/projects/github.com/cisco-ospo/oss-template/compare/15721c2f6610ef246a07666809ee148f93fb71bd/99689cce7148816cbaae4cd0d75e0de82f434869) | [View](https://ossf.github.io/scorecard-visualizer/#/projects/github.com/cisco-ospo/oss-template/commit/99689cce7148816cbaae4cd0d75e0de82f434869) | [Fix it](https://app.stepsecurity.io/securerepo?repo=cisco-ospo/oss-template) |
+| [cisco-ospo/fsoc](https://github.com/cisco-ospo/fsoc) | [c40a8a2](https://github.com/cisco-ospo/fsoc/commit/c40a8a2db11920f80a5e0fee2fb2fee68b7a2e66) | 6.7 | 2024-02-24T01:45:26Z | 0 / [Details](https://ossf.github.io/scorecard-visualizer/#/projects/github.com/cisco-ospo/fsoc/compare/c40a8a2db11920f80a5e0fee2fb2fee68b7a2e66/c40a8a2db11920f80a5e0fee2fb2fee68b7a2e66) | [View](https://ossf.github.io/scorecard-visualizer/#/projects/github.com/cisco-ospo/fsoc/commit/c40a8a2db11920f80a5e0fee2fb2fee68b7a2e66) | [Fix it](https://app.stepsecurity.io/securerepo?repo=cisco-ospo/fsoc) |
+| [cisco-ospo/oss-template-go](https://github.com/cisco-ospo/oss-template-go) | [fbb045c](https://github.com/cisco-ospo/oss-template-go/commit/fbb045cbab4920e93f8bb90ac0abfe4cdaa156bd) | 4.9 | 2024-05-11T01:45:57Z | 0 / [Details](https://ossf.github.io/scorecard-visualizer/#/projects/github.com/cisco-ospo/oss-template-go/compare/fbb045cbab4920e93f8bb90ac0abfe4cdaa156bd/fbb045cbab4920e93f8bb90ac0abfe4cdaa156bd) | [View](https://ossf.github.io/scorecard-visualizer/#/projects/github.com/cisco-ospo/oss-template-go/commit/fbb045cbab4920e93f8bb90ac0abfe4cdaa156bd) | [Fix it](https://app.stepsecurity.io/securerepo?repo=cisco-ospo/oss-template-go) |
 
-_Report generated by [UlisesGascon/openssf-scorecard-monitor](https://github.com/UlisesGascon/openssf-scorecard-monitor)._
\ No newline at end of file
+_Report generated by [UlisesGascon/openssf-scorecard-monitor](https://github.com/UlisesGascon/openssf-scorecard-monitor)._
```

Comment:
```suggestion
_Report generated by [OpenSSF Scorecard Monitor](https://github.com/ossf/scorecard-monitor)._
```

Reply:
Good catch! Can do 🫡

Reply:
Btw, looks like this was _partially_ [fixed upstream](https://github.com/ossf/scorecard-monitor/pull/75) already, but there was a find/replace error on the `issue.ejs` template: https://github.com/ossf/scorecard-monitor/pull/75/files#diff-32816260c1e1ed95d3ecb3128ec495d71816e45440ae2d0a189c139f762e91f8. 

Submitted upstream fix as https://github.com/ossf/scorecard-monitor/pull/83.